### ISSUE TITLE
Правки баланса

### ISSUE THREAD

**Issue by ToxaBes** [ToxaBes/ADR-Spec-Ops#149]
Если синие захватили КП/Бункер то через 5 мин ящики с оружием на КП исчезают. В этом случае партизаном не останется халявный арсенал.

**Comment by Horizon1959** [comment]
@ToxaBes Это его за 3 минуты будут заряжать?

**Comment by ToxaBes** [comment]
нет, он будет уходить из зоны патрулирования на свою базу (за границей карты) для пополнения БК. А с базы будет лететь свежий самолет. Ротационное патрулирование РБД.

**Comment by Horizon1959** [comment]
@ToxaBes сообщение об этом будет? Будет ли это касаться вражеских вертолётов?

**Comment by ToxaBes** [comment]
>сообщение об этом будет?

нет.

>Будет ли это касаться вражеских вертолётов?

да.

Я думаю немного иначе сделаем, не раз в 10 минут, а как отстреляет БК. После чего улетает на базу и в течение 5-10 мин прилетает "новый". Так будет реалистичней.

**Comment by Horizon1959** [comment]
@ToxaBes да, круто.

**Comment by 12st** [comment]
Весь бк он не потратит в любом случае. Поэтому надо делать, что-то вроде проверки на кол-во бк. Но не так, чтобы потратив 2 AA ракеты он сразу на перезарядку улетал.

**Comment by ToxaBes** [comment]
>Весь бк он не потратит в любом случае.

Мы можем проверять по БК все что не пулеметы. Если там пусто то пора на перезарядку.

**Comment by 12st** [comment]
@ToxaBes штурмовик никогда не скинет бомбы и редко стреляет не управляемыми.

**Comment by ToxaBes** [comment]
Тогда нужно будет по каждому виду собрать чем он стреляет в большинстве случаев и что будем проверять.

**Comment by 12st** [comment]
1)Штурмовик. По технике в первую очередь летят управляемые ПТ при их отсутствии используют неуправляемые ПП или пушку, по воздуху АА ракеты или пушку. По пехоте не стреляет. 
2)Касатка. По технике отстреливает ракеты, по пехоте пулемётами, но пулемёта очень много(2000)
3)Бузард. С пушки по технике. 6 АА ракет против воздуха или пушка. По пехоте не стреляет.
4)Кайман. С пушки по воздуху и пехоте. По технике скальпелями или ББ с пушки.

**Comment by ToxaBes** [comment]
@12st - спасибо за информацию. Можно сделать например так:

1. Штурмовик  - проверка управляемых ПТ + АА ракет
2. Касатка - проверка ракет
3. Бузард - проверка АА ракет
4. Кайман - проверка Скальпелей + ББ

Т.е. если у штурмовика кончились управляемые ПТ **и** АА ракеты то он уходит на перезарядку, если у Каймана кончились Скальпели **и** ББ - тоже на перезарядку и т.д.

**Comment by 12st** [comment]
@ToxaBes всё нормально кроме штурмовика, он будет очень быстро улетать тк это всего 4 ракеты пт  , а по воздуху он может и с пушек стрелять достаточно эффективно.

**Comment by ToxaBes** [comment]
Спасибо, поправил предыдущее сообщение.

**Comment by 12st** [comment]
Оказалось, что боты не умеют переключать тип боеприпасов. Поэтому кайман будет долбить по всему сначала с ОФ, а потом ББ в не зависимости от типа цели.

**Comment by ToxaBes** [comment]
Напиши, что в таком случае стоит проверять у Каймана.

**Comment by 12st** [comment]
Тут получается, что он пока пушку не кончится смысла ему на перезарядку лететь нет. Если мы про реализм говорим.

**Comment by ToxaBes** [comment]
>Тут получается, что пока пушка не кончится смысла ему на перезарядку лететь нет. Если мы про реализм говорим.

Нет, пушка не основное орудие и никто не будет летать пока не израсходует весь БК. Если мы про реализм говорим.

**Comment by 12st** [comment]
Тут получается так, ОФ мало эффективны против бронированных целей. То есть после того как он сделает 250 выстрелов у него снизится урон по пехоте, но возрастёт по воздуху и технике.

**Comment by ToxaBes** [comment]
Нам нужен простой алгоритм оставаться или улетать. Там же еще есть разделение пилот/стрелок, может по ним отдельно проверять. И кстати почему это боты не умеют переключать оружие? Боты в Каймане на раз переключаются на ББ и выносят броню на подъезде к основе.

**Comment by 12st** [comment]
Странно, сам решил проверить и запускал в редакторе. По вражескому танку летели именно фугасы.

**Comment by ToxaBes** [comment]
Какое в этом случае будем правило использовать для Каймана?

**Comment by Noart537** [comment]
Пушку и скальпели. Танки не так часто бывают. А фугасами он все летающее и легкое выносит на раз.
По танкам кайман старается птурами работать впервую очередь. Обычно он не успевает их потратить, его быстрее сбивают.

**Comment by ToxaBes** [comment]
Ок, исправил.
